test/testAll.m

```diff
@@ -22,12 +21,13 @@
     CBTDIR = pth(1:end-(length('testAll.m') + 1));
 
     % change the directory to the root
-    cd([CBTDIR, filesep, '..', filesep]);
-
-    % include the root folder and all subfolders
-    addpath(genpath(pwd));
+    cd([CBTDIR, filesep, '..', filesep]);    
+else
+    CBTDIR = fileparts(which('initCobraToolbox.m'));
+    cd(CBTDIR);
 end
-
+%include the root folder and all subfolders.
+addpath(genpath(pwd));
 CBTDIR = fileparts(which('initCobraToolbox.m'));
```

Comment:
agreed, but the assignment of CBTDIR to the test folder was not my code ^^ 
I'll change it anyways. (along with folder changes)